_rules/SC1-1-1-accessible-name-is-image-filename.md

```diff
@@ -0,0 +1,89 @@
+---
+name: image accessible name is image filename
+description: |
+  This rule checks that `img` elements that use their source filename as their accessible name, do so without loss of infomation to the user.
+
+success_criterion:
+- 1.1.1 # Non-Text Content
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Bryn Anderson
+---
+
+## Test procedure
+
+### Applicability
+
+The rule applies to HTML `input type="image" `and `area` elements or any HTML element with the semantic role of `img` [exposed to assistive technologies](#exposed-to-assistive-technologies), where the [accessible name](#accessible-name) value exactly matches the filename specified in the `src` attribute.
+
+**Note**: URL query strings, case sensitivey, forward and trailing whitespace should be ignored. For example `src="/foo/bar.jpg?baz "` has a file name "bar.jpg".
```

Comment:
And to be consistent, replace "file name" by "filename" in the note

> **Note**: URL query strings, case sensitivey, forward and trailing whitespace should be ignored. For example `src="/foo/bar.jpg?baz "` has a filename "bar.jpg".

Reply:
And a typo (I think?) : 

"case sensitivey" should be "case sensitive"

Reply:
Thanks for spotting the typo's and inconsistency @audreymaniez .

In regards to the filename extension, the intention was that the rule checks for an exact match to avoid catching instances of e.g. `src="/foo/Deque.jpg" alt="Deque"` which I would argue should not be flagged by default.
Which would mean....
- `src="/foo/bar_2.jpg" alt="bar_2.jpg"` is applicable.
- `src="/foo/bar_2" alt="bar_2"` is applicable.
- `src="/foo/bar_2.jpg" alt="bar_2"` is NOT applicable.